backend/backends/stdout/stdout.go

```diff
@@ -79,14 +79,14 @@ func (client *Client) SendMetrics(metrics types.MetricMap) error {
 		fmt.Fprintf(buf, "stats.set.%s %d %d\n", nk, len(set.Values), now)
 	})
 
-	fmt.Fprintf(buf, "statsd.numStats %d %d\n", metrics.NumStats, now)
-	fmt.Fprintf(buf, "statsd.processingTime %f %d\n", float64(metrics.ProcessingTime)/float64(time.Millisecond), now)
-
-	_, err := buf.WriteTo(log.StandardLogger().Writer())
-	if err != nil {
-		return err
-	}
-	return nil
+	writer := log.StandardLogger().Writer()
+	defer func() {
+		if err := writer.Close(); err != nil && retErr == nil {
```

Comment:
The problem is not about the verbosity in the logs, it's about the verbosity in the code ;) We will probably never go and look at the errors when statsd is closed but we'll keep the verbosity in the code forever.